roles/openshift_storage_nfs/tasks/main.yml

```diff
@@ -1,31 +1,41 @@
 ---
-- name: Set nfs facts
-  openshift_facts:
-    role: nfs
-    local_facts:
-      exports_dir: "{{ openshift_nfs_exports_dir | default(None) }}"
-      export_options: "{{ openshift_nfs_export_options | default(None) }}"
-      registry_volume: "{{ openshift_nfs_registry_volume | default(None) }}"
-
 - name: Install nfs-utils
   yum:
     pkg: nfs-utils
     state: present
 
+- name: Configure NFS
+  lineinfile:
+    dest: /etc/sysconfig/nfs
+    regexp: "{{ item.regex }}"
+    line: "{{ item.line }}"
+  register: nfs_config
+  with_items:
+  - regex: '^RPCNFSDARGS=.*$'
+    line: 'RPCNFSDARGS="-N 2 -N 3"'
+  - regex: '^RPCMOUNTDOPTS=.*$'
+    line: 'RPCMOUNTDOPTS="-p 20048"'
+  - regex: '^STATDARG=.*$'
+    line: 'STATDARG="-p 50825"'
+
+- name: Restart nfs-config
+  service: name=nfs-config state=restarted
+  when: "True in (nfs_config.results | oo_collect(attribute='changed') | list)"
```

Comment:
Soes `nfs_config | changed` not work?

Also, the RPCMOUNTDOPTS and STATDARG should not be needed now